aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -49,39 +49,32 @@ module aptos_framework::account {
 
     const MAX_U64: u128 = 18446744073709551615;
 
-    /// Account already existed
-    const EACCOUNT: u64 = 0;
-    /// Sequence number exceeded the maximum value for a u64
-    const ESEQUENCE_NUMBER_TOO_BIG: u64 = 1;
-    /// The address provided didn't match the `aptos_framework` address.
-    const ENOT_APTOS_FRAMEWORK: u64 = 2;
+    /// Account already existed.
+    const EACCOUNT_ALREADY_EXISTS: u64 = 1;
+    /// Sequence number exceeded the maximum value for a u64.
+    const ESEQUENCE_NUMBER_TOO_BIG: u64 = 2;
     /// The provided authentication had an invalid length
```

Comment:
`The provided authentication key has an invalid length`

Reply:
Updated